server/README.md

```diff
@@ -0,0 +1,321 @@
+# BraveSensor-Server
+
+[![Build Status](https://travis-ci.com/bravetechnologycoop/BraveSensor-Server.svg?branch=master)](https://travis-ci.com/bravetechnologycoop/BraveSensor-Server)
+
+# How to release a new version of BraveSensor-Server
+
+## 1. Update git to reflect the new release
+
+1. on your local machine, in the `BraveSensor-Server` repository:
+
+   1. pull the latest code ready for release: `git checkout devenv && git pull origin devenv`
+
+   1. decide on an appropriate version number for the new version
+
+   1. update `CHANGELOG.md` by moving everything in `Unreleased` to a section for the new version
```

Comment:
If we do separate tags, then I guess this `CHANGELOG.md` can move into `/server` and a new one would be needed in `/firmware`.